### PR TITLE
Minor clarification

### PR DESCRIPTION
**Pull request by drewnoakes**
Selecting "Add Item..." seems like a more natural action to take to create an extensibility item in the project.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@drewnoakes : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

File: docs/extensibility/extensibility-hello-world.md [hunk 1]
```diff
@@ -128,4 +128,4 @@ Now that you know the basics of working with Visual Studio Extensibility, here's
 
 * [Start to develop Visual Studio extensions](starting-to-develop-visual-studio-extensions.md) - Samples, tutorials. and publishing your extension.
 * [What's new in the Visual Studio 2017 SDK](what-s-new-in-the-visual-studio-2017-sdk.md) -New extensibility features in Visual Studio 2017
-* [Inside the Visual Studio SDK](internals/inside-the-visual-studio-sdk.md) - Learn the details of Visual Studio Extensibility
\ No newline at end of file
+* [Inside the Visual Studio SDK](internals/inside-the-visual-studio-sdk.md) - Learn the details of Visual Studio Extensibility
```
Comment (drewnoakes) [inline docs/extensibility/extensibility-hello-world.md]: EOF was changed by the web editor.

**Comment by PRMerger6** [comment]
@gewarren : Thanks for your contribution! The author, @gregvanl, has been notified to review your proposed change.

**Comment by drewnoakes** [comment]
Great. Thanks Genevieve.

**Pull request merged**